src/Compilers/Core/Portable/CodeGen/LocalConstantDefinition.cs

```diff
@@ -17,22 +16,26 @@ internal sealed class LocalConstantDefinition : Cci.ILocalDefinition
         private readonly string _name;
         private readonly Location _location;
         private readonly Cci.IMetadataConstant _compileTimeValue;
-        private readonly bool _isDynamic;
 
         //Gives the synthesized dynamic attributes of the local definition
         private readonly ImmutableArray<TypedConstant> _dynamicTransformFlags;
-
-        public LocalConstantDefinition(string name, Location location, Cci.IMetadataConstant compileTimeValue, bool isDynamic = false,
-            ImmutableArray<TypedConstant> dynamicTransformFlags = default(ImmutableArray<TypedConstant>))
+        private readonly ImmutableArray<TypedConstant> _tupleElementNames;
```

Comment:
Consider removing these two backing fields, and instead just use the readonly properties (`DynamicTransformFlags` and `TupleElementNames`).